pynicotine/pynicotine.py

```diff
@@ -141,6 +141,7 @@ def start(self, ui_callback, network_callback):
 
         self.transfers.init_transfers()
         self.privatechats.load_users()
+        self.userlist.load_users()
```

Comment:
Removing this line doesn't make any apparent difference, the buddy list is still populated even when auto-connect at startup is disabled.

This suggests that some unnecessary save_list() and update() calls are being done during the startup sequence in the userlist module.